### PR TITLE
Fix Typos, Add Terms, and Apply Sentence Case for Improved Documentation

### PR DESCRIPTION
**Pull request by Sonamla1797**
Title:
"Enhance Documentation Quality: Fix Typos, Add Terms, Apply Sentence Case, and Configure Styles for Microsoft Terminology"

Description:
This pull request introduces several enhancements to the documentation:

Fix Typos:

Addressed various typographical errors found in the documentation, promoting accuracy and readability.
Add Terms to Accept.txt:

Included terms in the accept.txt file that previously triggered errors during the vale checks, ensuring a smoother validation process.
Apply Sentence Case for Headings:

Adjusted all headings to use sentence case uniformly, enhancing the consistency and professionalism of the documentation.
Configure Styles for Microsoft Terminology:

Added a Microsoft folder with a .vale.ini file inside the styles directory to configure specific styles for Microsoft terminology. This helps prevent errors that occurred during the vale command execution.
These changes collectively aim to elevate the quality of the documentation and align with the project's guidelines. The addition of the Microsoft folder with a tailored .vale.ini file further refines the documentation for a more comprehensive and error-free experience.

Your feedback on these improvements is highly valued. Please review the changes, and if there are any additional adjustments or considerations, feel free to let me know. Thank you for your time and consideration!

### CODE REVIEW

**Comment by stevepiercy** [comment]
@Sonamla1797 please read and follow [Contributing to Plone](https://6.docs.plone.org/contributing/index.html) and [First-time contributors](https://6.docs.plone.org/contributing/first-time.html). Once you receive confirmation that you have been added to the Plone Contributors team, then post a comment here to notify us that we can start a review. Thank you!

**Comment by Sonamla1797** [comment]
Hi @stevepiercy ,
Just to inform you that i have been added to the Plone contributors team would proceed with the review.
Thank you

File: docs/source/client/actions/addons.md
Comment (stevepiercy) [inline docs/source/client/actions/addons.md:1]: All the terms should not use `"` or `'`, but backticks, to make then inline literals. For example:

```
- `addonId`: string
```

File: docs/source/client/actions/comments.md
Comment (stevepiercy) [inline docs/source/client/actions/comments.md:1]: Similar to other file, all the terms should not use " or ', but backticks, to make then inline literals.

File: docs/source/client/actions/content.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Use the `useGetContent` hook to get the content at a given path.
 
   - **Required:** No
 
-- **fullObjects:** boolean
+- 'fullObjects:' boolean
```
Comment (stevepiercy) [inline docs/source/client/actions/content.md:29]:
```suggestion
- `fullObjects`: boolean
```

File: docs/source/client/actions/lock.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Use the `useUpdateLock` hook to update a lock for a page.
 
   - **Required:** Yes
 
-- **locktoken**: string
+- 'locktoken': string
```
Comment (stevepiercy) [inline docs/source/client/actions/lock.md:73]:
```suggestion
- `locktoken`: string
```

File: docs/source/client/actions/querystringsearch.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ Use the `useGetQuerystringSearch` hook to get the search results for a page.
 
   - **Required**: No
 
-- **fullobjects**: boolean
+- **fullObjects**: boolean
```
Comment (stevepiercy) [inline docs/source/client/actions/querystringsearch.md:57]:
```suggestion
- `fullObjects`: boolean
```

File: docs/source/client/actions/querystringsearch.md [hunk 4]
```diff
@@ -120,7 +120,7 @@ Use the `usePostQuerystringSearch` hook to get the search results for a page.
       - **Required**: No
       - The order to sort on.
 
-    - `fullobjects: boolean`
+    - `fullObjects: boolean`
```
Comment (stevepiercy) [inline docs/source/client/actions/querystringsearch.md:123]:
```suggestion
    - `fullObjects`: boolean
```

File: docs/source/client/actions/relations.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ Use the `useGetRelations` hook to get the relations for the given parameters.
 
   - **Required**: No
 
-- **onlyBroken**: boolean
+- 'onlyBroken': boolean
```
Comment (stevepiercy) [inline docs/source/client/actions/relations.md:43]:
```suggestion
- `onlyBroken`: boolean
```

File: docs/source/client/actions/rules.md [hunk 6]
```diff
@@ -28,7 +28,7 @@ Use the `useCreateRule` hook to add a content-rule to a page.
 
 ### Parameters
 
-- **ruleId**: string
+- 'ruleId': string
```
Comment (stevepiercy) [inline docs/source/client/actions/rules.md:31]:
```suggestion
- `ruleId`: string
```

File: docs/source/client/actions/transactions.md [hunk 7]
```diff
@@ -31,6 +31,6 @@ Use the `useRevertTransactions` hook to revert transactions in the portal.
   - **Required**: Yes
   - It can have the following fields:
 
-    - **transactionIds**: string[]
+    - 'transactionIds': string[]
```
Comment (stevepiercy) [inline docs/source/client/actions/transactions.md:34]:
```suggestion
    - `transactionIds`: string[]
```

File: docs/source/client/actions/types.md [hunk 8]
```diff
@@ -24,11 +24,11 @@ Use the `useGetType` hook to get the information about the content type provided
 
 ### Parameters
 
-- **contentPath**: string
+- 'contentPath': string
```
Comment (stevepiercy) [inline docs/source/client/actions/types.md:27]:
```suggestion
- `contentPath`: string
```

File: docs/source/client/actions/types.md [hunk 9]
```diff
@@ -56,7 +56,7 @@ Use the `useCreateTypeField` hook to add a field/fieldset to the type provided.
...
```
Comment (stevepiercy) [inline docs/source/client/actions/types.md:59]:
```suggestion
- `contentPath`: string
```

File: docs/source/client/actions/types.md [hunk 10]
```diff
@@ -93,7 +93,7 @@ Use the `useUpdateTypeField` hook to update a field/fieldset of the type provide
...
```
Comment (stevepiercy) [inline docs/source/client/actions/types.md:96]:
```suggestion
- `contentPath`: string
```

File: docs/source/client/actions/users.md [hunk 11]
```diff
@@ -19,7 +19,7 @@ Use the `useGetUsers` hook to get the list of all users in the portal.
 
   - **Required**: No
 
-- **groupsFilter**: string[]
+- 'groupsFilter': string[]
```
Comment (stevepiercy) [inline docs/source/client/actions/users.md:22]:
```suggestion
- `groupsFilter`: string[]
```

File: docs/source/client/future-improvements.md [hunk 12]
```diff
@@ -34,7 +34,7 @@ This would mean that anyone who wants to use `@plone/client` would need to insta
 
 1. mocking direct dependencies of projects is much easier. We can then mock `axios` directly instead of using `nock`, although the latter has no clear downsides either
 2. the projects that wants to use `@plone/client` can bring their own `@tanstack/react-query` - otherwise they would need to import the `@tanstack/react-query` dependency indirectly from `@plone/client` and then cannot have their own version of it without conflicts - same applies to any project that already uses `axios` as well
-3. projects using other frameworks like vue would need to install `@tanstack/vue-query` anyway, so we need to have some consistency around frameworks, thus it makes sense that projects with react also bring their own version of the dependency
+3. projects using other frameworks like Vue would need to install `@tanstack/Vue-query` anyway, so we need to have some consistency around frameworks, thus it makes sense that projects with react also bring their own version of the dependency
```
Comment (stevepiercy) [inline docs/source/client/future-improvements.md:37]:
```suggestion
3. projects using other frameworks like Vue would need to install `@tanstack/vue-query` anyway, so we need to have some consistency around frameworks, thus it makes sense that projects with React also bring their own version of the dependency
```

File: docs/source/client/quick-start.md [hunk 13]
```diff
@@ -13,12 +13,12 @@ It provides the artifacts that TanStack Query requires to work:
 The API request layer allows to build and send arbitrary requests to Plone REST API endpoints.
 It has the potential to also be able to send requests to other APIs (provided the custom Query options factories/functions).
 
-The Javascript Plone client is conceived to work with TanStack Query, the query or mutation functions can be used to call any Plone REST API endpoint without using it.
-These functions can be used in other use cases like command line helpers, scripts or programatic tests.
+The JavaScript Plone client is conceived to work with TanStack Query, the query or mutation functions can be used to call any Plone REST API endpoint without using it.
+These functions can be used in other use cases like command line helpers, scripts or programmatic tests.
```
Comment (stevepiercy) [inline docs/source/client/quick-start.md:17]:
```suggestion
These functions can be used in other use cases, like command line helpers, scripts, or programmatic tests.
```

File: styles/Vocab/Base/accept.txt
Comment (stevepiercy) [inline styles/Vocab/Base/accept.txt:1]: Delete this file. This is not how to configure custom words in Plone. See [Advanced Vale usage](https://6.docs.plone.org/contributing/documentation/authors.html#advanced-vale-usage).

Additionally, Vale will ignore inline literals (if you wrap them with backticks, of course), and most of these are inline literals. Additionally some of these terms should already exist, and they should always be sorted alphabetically.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Mostly use backticks ` instead of quotes around terms. Otherwise this is pretty good,

**Pull request closed without merging**